packages/admin/src/components/DitoRoot.vue

```diff
@@ -166,7 +167,8 @@ export default DitoComponent.component('dito-root', {
           },
           password: {
             type: 'password'
-          }
+          },
+          ...extraComponents || {}
```

Comment:
You don't need the `|| {}` here, it works without it, you can expand `null` into objects, but not into arrays.